common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -14,6 +14,16 @@ trait FeatureSwitches {
     exposeClientSide = false
   )
 
+  val HeroicTemplateSwitch = Switch(
```

Comment:
maybe a comment in here referring to this PR for anyone removing the switch. there's a quite a bit of bespoke stuff going in, which if left behind and if you don't what it is, you wouldn't want to remove later